test/http-api/inject/test-version.js

```diff
@@ -17,9 +17,9 @@ module.exports = (http) => {
         method: 'GET',
         url: '/api/v0/version'
       }, (res) => {
-        expect(res.result.version).to.equal(pkgversion)
-        expect(res.result).to.have.a.property('commit')
-        expect(res.result).to.have.a.property('repo')
+        expect(res.result).to.have.a.property('Version', pkgversion)
```

Comment:
that's exactly what is happening here, `pkgversion` is read from `package.json` which is the version that js-ipfs uses as baseline when answering the version command.